### ISSUE TITLE
Enable identification of starting 11 for each game

### ISSUE THREAD

**Issue by dcaribou** [dcaribou/transfermarkt-datasets#127]
Reported in Kaggle [here](https://www.kaggle.com/datasets/davidcariboo/player-scores/discussion/370339).

**Comment by dcaribou** [comment]
We've found that there are limitations with the approach above to find starting lineups.

To use the approach above (using game events to reconstruct lineups) we rely on accurate appearance information. In particular, we need to know if a player has a appearance on a particular game and for which team it was, so we can apply substitution events to reconcile starting lineups. However we've found that some appearances seem to contains inaccurate information in this regard. Consider the following case:

Vitaliy Lysytskyi (player ID 24032) was part of the starting lineup of Metalurg Zaporizhya on Sat, 8/2/14, according to the [game sheet](https://www.transfermarkt.co.uk/metalurg-zaporizhya-2016-_goverla-uzhgorod-2016-/index/spielbericht/2477955). However, according to his [record of games](https://www.transfermarkt.co.uk/vitaliy-lysytskyi/leistungsdatendetails/spieler/24032/plus/0/verein/6994), he played that game for Goverla Uzhgorod.

Although it does seem like the latter is the error, as he played the rest of the games of that season for Metalurg Zaporizhya, it would be tricky to define an algorithm that corrects these inconsistencies. A better approach may be to directly rely on lineup information from the games sheet (https://github.com/dcaribou/transfermarkt-scraper/issues/59)
